.env.example

```diff
@@ -1,7 +1,10 @@
-APP_PORT=3000
-APP_HOST=localhost
-APP_NAME=MyApp
-POSTGRES_HOST=db.example.com
+BACKEND_PORT=3333
+BACKEND_HOST=localhost
+BACKEND_NAME=MyApp
+POSTGRES_HOST=localhost
```

Comment:
Could u add some general inline docs information about the configs f.e. POSTGRES, Backend etc and add spaces.